lizard_auth_client/views.py

```diff
@@ -844,8 +877,33 @@ def form_valid(self, form):
         # portal that initiated the creation of the new SSO account. This may
         # vary since the server can have multiple hostnames.
         payload['visit_url'] = self.request.get_host()
-        response = _sso_post('new-user', payload)
-        updated_values = response['user']
+        try:
+            json_data = _new_user_sso_post('new-user', payload)
+        except requests.exceptions.HTTPError as e:
+            json_data = e.response.json()
+
+            if e.response.status_code == 409:
+                form.add_error(
+                    'username',
+                    'This username is already in use, please specify a '
+                    'different one')
+                return self.form_invalid(form)
+            elif e.response.status_code == 200:
```

Comment:
200, dat is geen error. Dan kom je toch nooit hier in de except hierbij terecht?